README.md

```diff
@@ -162,6 +162,13 @@ Regarding the test project, one thing that's important is to name the test proje
   - This is for improving CI performance so to find exactly one copy of your test assembly.
   - Also, due to test dependencies, the test project should build both .NET 4.5.2 and NETStandard 1.4. For example, check out "src\SDKs\Resource\Resource.tests"
 
+### Tested OSs and .NET Versions
+
+|                        | Linux (Ubuntu 16.04) | MacOS 10.13 | Windows Server 2016 |
+|------------------------|----------------------|-------------|---------------------|
+| **.NET Core 2.1.503**  |  x                   |      x      |          x          |
```

Comment:
2.1.503 is actually the SDK version. The real version of .NET Core is 2.1. We are currently adding .NET Framework 4.6.1 support as part of https://github.com/Azure/azure-sdk-for-net/issues/5367